Tribler/community/market/community.py

```diff
@@ -1343,7 +1359,7 @@ def received_order_status(self, _, payload):
             "traded_quantity": payload.traded_quantity.amount,
             "timeout": float(payload.timeout),
             "timestamp": float(payload.timestamp),
-            "ip": payload.address.ip,
+            "address": payload.address.ip,
```

Comment:
IP addresses (+ports) are not stable. Your own address is just an estimation of whatever the interface address of your most connected-to NAT box is at that time. This especially holds for port stability, which may be reassigned as needed.

Even worse would be if you were using this on a mobile device and, god forbid, performing some form of physical movement.

Apparently this was already in the code, but I just noticed 😃

Reply:
We discussed this 👍